src/app/components/applications/SSV/MyAccount/components/Withdraw/OperatorFlow.tsx

```diff
@@ -22,10 +25,15 @@ const OperatorFlow = ({ operator }: { operator: IOperator; }) => {
   const classes = useStyles();
   const operatorBalance = operator.balance ?? 0;
   const dispatch = useAppDispatch();
+  const stores = useStores();
+  const processStore: ProcessStore = stores.Process;
+  const process: SingleOperator | SingleCluster = processStore.getProcess;
 
   const withdrawSsv = async () => {
     setIsLoading(true);
     const success = await withdrawRewards({ operator, amount: inputValue.toString(), isContractWallet, dispatch });
+    const balance = await getOperatorBalance({ id: operator.id });
```

Comment:
please check why after fetching all operators data it is not getting here
don't use process store please

Reply:
the data which came from API does not have balance,
we get balance from contract

Reply:
done